pages/docs/manual/latest/pipe-last.mdx

```diff
@@ -0,0 +1,22 @@
+# Pipe Last
+
+`|>` is similar to [pipe first (`->`)](./pipe-first) but it sends the left side value to the last argument of the right side. `a(b, c)` becomes `c|>a(b)`.
+
+Note that pipe last has a runtime cost due to runtime currying.
+
+### Tips & Tricks
+
+For bucklescript codebases, try to use pipe first over pipe last whenever it makes sense. 
+
+That helps the compiler to eliminate runtime currying and it's also easier to map to JS apis due to their t-first nature
```

Comment:
For the rest of this 'Tips and Tricks' section, I would suggest to not cover the advantages of pipe-first, or placeholders, since those are already covered in the pipe-first doc page. Instead maybe a para about when you _would_ want to use pipe-last (for code that's intended to be portable or more functional-style).

Reply:
@yawaramin I have to admit that I don't know enough about when pipe-last would be desirable. If you don't mind, could you add a commit for that paragraph?

Reply:
Not much to add 🙂 maybe just:

> You may come across or use the pipe-last operator in certain situations like when using libraries which use it, or when writing libraries intended to be portable to native Reason/OCaml.